packages/studio/bin/studio.ts

```diff
@@ -32,6 +33,7 @@ cli
         env: {
           ...process.env,
           YEXT_STUDIO_ARGS: JSON.stringify(options),
+          VITE_STUDIO_STRICT: String(useStrictMode),
```

Comment:
Are the possible values for `VITE_STUDIO_STRICT` just the strings "true" and "false", or could it also be "undefined" in the case where `options["--"]` above is undefined?

Reply:
Whether or not the `--` flag is specified/has arguments, `options ["--"]`  always returns an array. Either an empty one, or one with values.  So `options ["--"]` should always be defined.  I changed the type of `CliArgs` to reflect that `--` should never be undefined!